scenarioo-client/app/manage/generalSettings/generalSettings.html

```diff
@@ -54,6 +54,17 @@
             </div>
         </div>
 
+        <div class="row sc-space-bottom">
+            <div class="col-lg-3">Search engine
+                &nbsp;<span class="glyphicon glyphicon-info-sign" tooltip="Application specific full text search engine."></span>
+            </div>
+            <div class="col-lg-9" ng-show="!vm.isEngineRunning" ng-cloak>
+                <div class="alert alert-warning">No search engine running</div>
+            </div>
+            <div class="col-lg-9" ng-show="vm.isEngineRunning" ng-cloak>
+                <span>up and running!</span>
```

Comment:
on the demo I see this text, but no search field for searching - is that as expected?